extensions/blocks/image-compare/image-compare.php

```diff
@@ -54,7 +54,20 @@ function load_assets( $attr, $content ) {
  * @return string
  */
 function render_amp( $attr ) {
-	// @TODO: what to show for AMP view?
-	// :shrug-emoji:
-	return '';
+	$img_before_id  = $attr['imageBeforeId'];
+	$img_before_url = $attr['imageBeforeUrl'];
+	$img_before_alt = $attr['imageBeforeAlt'];
+	$img_after_id   = $attr['imageAfterId'];
+	$img_after_url  = $attr['imageAfterUrl'];
+	$img_after_alt  = $attr['imageAfterAlt'];
+
+	return sprintf(
+		'<amp-image-slider layout="responsive" width="300" height="200"> <amp-img id="%d" src="%s" alt="%s" layout="fill"></amp-img> <amp-img id="%d" src="%s" alt="%s" layout="fill"></amp-img></amp-image-slider>',
+		esc_attr( $img_before_id ),
+		esc_attr( $img_before_url ),
+		esc_attr( $img_before_alt ),
+		esc_attr( $img_after_id ),
+		esc_attr( $img_after_url ),
+		esc_attr( $img_after_alt )
```

Comment:
What do you think about using numbered placeholders and more specific escaping here?

```suggestion
		'<amp-image-slider layout="responsive" width="300" height="200"> <amp-img id="%1$d" src="%2$s" alt="%3$s" layout="fill"></amp-img> <amp-img id="%4$d" src="%5$s" alt="%6$s" layout="fill"></amp-img></amp-image-slider>',
		absint( $img_before_id ),
		esc_url( $img_before_url ),
		esc_attr( $img_before_alt ),
		absint( $img_after_id ),
		esc_url( $img_after_url ),
		esc_attr( $img_after_alt )
```